### DISCUSSION TITLE
Dropping unsynced node during fast sync

### DISCUSSION THREAD

**Discussion by cxhjmfl** [kcc-community/any-advice-issue#7] [General]
Dropping unsynced node during fast sync  id=c7e7675577416d6b conn=dyndial addr=52.176.100.77:30303 type=Parity-Ethereum/v2.5.11-stable-fc129b4-20191205/x86_64-linux-gnu/rustc1.39.0

**Comment by viaweb3** [comment]
please comment more message here.

**Reply by viaweb3** [reply]
I think there is no Parity client in KCC network, what's your command?
